inc/views/template_parts.php

```diff
@@ -145,6 +145,13 @@ public function render_featured_post() {
 		$wrapper_classes  = apply_filters( 'neve_posts_wrapper_class', [] );
 		$posts_to_exclude = [];
 
+		$default_ordered_components = array(
+			'thumbnail',
+			'title-meta',
+			'excerpt',
+		);
+		$ordered_components         = json_decode( get_theme_mod( 'neve_post_content_ordering', wp_json_encode( $default_ordered_components ) ) );
```

Comment:
This can be extracted to a separate function and can be reused to keep the code DRY.